shared/js/background/chrome-events.es6.js

```diff
@@ -200,6 +218,27 @@ chrome.runtime.onMessage.addListener((req, sender, res) => {
         res(pixel.fire.apply(null, fireArgs))
         return true
     }
+
+    if (req.checkThirdParty) {
+        var action = {
+            isThirdParty: false,
+            shouldBlock: false
+        }
+        const tab = tabManager.get({ tabId: sender.tab.id })
+        if (tab && tab.site.whitelisted) {
+            res(action)
+        }
+
+        if (!utils.isFirstParty(req.frameUrl, sender.tab.url)) {
```

Comment:
Do we need to get the `frameUrl` in the content script? According to [MDN](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/MessageSender) the `sender.url` will return the frame URL if it is sent from a frame:

> If the sender is a script running in a web page (including content scripts as well as normal page scripts), then url will be the web page URL. If the script is running in an iframe, url will be the iframe's URL.